### PR TITLE
posthog relief x3

### PR DESCRIPTION
**Pull request by chitalian**

### CODE REVIEW

File: valhalla/jawn/src/middleware/auth.ts [hunk 1]
```diff
@@ -19,37 +20,16 @@ export const logInPostHog = (
   const start = Date.now();
   const postHogClient = newPostHogClient();
   postHogClient?.capture({
-    distinctId: authParams?.organizationId ?? "unknown",
```
Comment (fuziontech) [inline valhalla/jawn/src/middleware/auth.ts:22]: you can leave this part - we don't do anything transactional here so you can go nuts

File: valhalla/jawn/src/middleware/auth.ts [hunk 2]
```diff
@@ -19,37 +20,16 @@ export const logInPostHog = (
...
-  if (authParams?.organizationId && postHogClient) {
-    try {
-      postHogClient.groupIdentify({
-        groupType: "organization",
-        groupKey: authParams.organizationId,
-      });
-    } catch (error) {
-      console.error("Error identifying organization in PostHog:", error);
-    }
-  }
```
Comment (fuziontech) [inline valhalla/jawn/src/middleware/auth.ts:45]: 100% this shouldn't be called super frequently

File: valhalla/jawn/src/middleware/auth.ts [hunk 3]
```diff
@@ -19,37 +20,16 @@ export const logInPostHog = (
...
-
   const onFinish = async () => {
     const duration = Date.now() - start;
 
     try {
       postHogClient?.capture({
-        distinctId: authParams?.organizationId ?? "unknown",
```
Comment (fuziontech) [inline valhalla/jawn/src/middleware/auth.ts:52]: same here, on capture totally cool to leave this in

**Review by fuziontech — APPROVED** [review]

File: valhalla/jawn/src/middleware/auth.ts [hunk 4]
```diff
@@ -19,37 +20,16 @@ export const logInPostHog = (
   const start = Date.now();
   const postHogClient = newPostHogClient();
   postHogClient?.capture({
-    distinctId: authParams?.organizationId ?? "unknown",
+    distinctId: uuid(),
```
Comment (chitalian) [inline valhalla/jawn/src/middleware/auth.ts:23]:
```suggestion
    distinctId: authParams?.organizationId ?? "unknown",
```

File: valhalla/jawn/src/middleware/auth.ts [hunk 5]
```diff
@@ -19,37 +20,16 @@ export const logInPostHog = (
...
-    }
-  }
-
   const onFinish = async () => {
     const duration = Date.now() - start;
 
     try {
       postHogClient?.capture({
-        distinctId: authParams?.organizationId ?? "unknown",
+        distinctId: uuid(),
```
Comment (chitalian) [inline valhalla/jawn/src/middleware/auth.ts:32]:
```suggestion
        distinctId: authParams?.organizationId ?? "unknown",
```

**Pull request merged**